### PR TITLE
Index list items

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
No re-index as previous reindex hasn't been triggered yet.

File: src/Search/Workflow/Blocks.php [hunk 1]
```diff
@@ -23,6 +23,18 @@
             $block->question ?? null,
             $this->flattenBlocks($block->answer ?? []),
             $this->flattenBlocks($block->content ?? []),
+            $this->flattenItems($block->items ?? []),
```
Comment (giorgiosironi) [inline src/Search/Workflow/Blocks.php]: ok, `"prefix": "roman-lower"` is excluded

Comment (giorgiosironi) [reply src/Search/Workflow/Blocks.php]: there can be HTML in `"text"` such as `"<i>P. khe59svzdsh80x9qhhhzon6m6fqqo2</i>"`. Maybe not a problem

Comment (thewilkybarkid) [reply src/Search/Workflow/Blocks.php]: Can be HTML already in lots of places. We could probably strip it.

**Pull request merged**